tests/client/chain/grpc/test_chain_grpc_exchange_v2_api.py

```
@@ -2647,6 +2647,116 @@
 
         assert orderbook == expected_orderbook
 
+    @pytest.mark.asyncio
+    async def test_fetch_market_balance(
+        self,
+        exchange_servicer,
+    ):
+        market_balance = exchange_query_pb.MarketBalance(
```

Pylint (3.3.7)

[error] Module 'pyinjective.proto.injective.exchange.v2.query_pb2' has no 'MarketBalance' member

(E1101)